ttnn/cpp/ttnn/operations/data_movement/copy/copy.cpp

```diff
@@ -0,0 +1,86 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+
+#include "ttnn/common/constants.hpp"
+#include "ttnn/run_operation.hpp"
+#include "ttnn/decorators.hpp"
+#include "ttnn/operations/data_movement/copy/copy.hpp"
+#include "device/copy_device_operation.hpp"
+
+namespace ttnn::operations::data_movement {
+
+ttnn::Tensor CopyOperation::operator()(
+    uint8_t queue_id,
+    const Tensor& src_tensor,
+    const Tensor& dst_tensor) {
+
+    std::vector<Tensor> dummy_outputs = {Tensor(operation::get_workers_for_op_output({src_tensor}))};
+    operation::launch_op(
```

Comment:
I think it can be removed, just register with auto launch, decorator will add it inside

Reply:
Done.